### ISSUE TITLE
Support for the Allrecipes redesign

### ISSUE THREAD

**Issue by lachlanjc** [shivamd/allrecipes#3]
I tried to get the allrecipes gem up and running, but it wasn't working. I just realized that Allrecipes [recently did a complete redesign](http://dish.allrecipes.com/calling-all-cooks/), so the HTML structure has been completely rewritten and requests from this gem aren't turning up any data. 😰

**Comment by shivamd** [comment]
@lachlanjc Thanks for the notice. I will try and get to this, this weekend and keep you posted.

**Comment by lachlanjc** [comment]
Thanks for looking into it! :v:

**Comment by shivamd** [comment]
I unfortunately do not have time to maintain this project. If anyone wants to go ahead and take over, please do so.

**Comment by ghost** [comment]
I am willing to take a shot at maintaining this project. I was looking to use it and am sad to see it's not currently working.

**Comment by shivamd** [comment]
@g-toledo4 Yup at this point, they changed their markup on the pages. So the code doesn't work.
